gdnative-core/src/nativescript/init.rs

```diff
@@ -270,6 +281,17 @@ impl<C: NativeClass> ClassBuilder<C> {
         }
     }
 
+    #[inline]
+    pub fn add_method_with_rpc_mode(&self, name: &str, method: ScriptMethodFn, rpc_mode: RpcMode) {
+        self.add_method_advanced(ScriptMethod {
+            name,
+            method_ptr: Some(method),
+            attributes: ScriptMethodAttributes { rpc_mode },
+            method_data: ptr::null_mut(),
+            free_func: None,
+        });
+    }
+
     #[inline]
     pub fn add_method(&self, name: &str, method: ScriptMethodFn) {
```

Comment:
Since there is a `add_method_with_rpc_mode` now, it might be a good idea to make `add_method` delegate to `add_method_with_rpc_mode` by calling `self.add_method_with_rpc_mode(name, method, RpcMode::Disabled)`.

Reply:
will do